docs/migrate-from-authn-to-idx.md

```diff
@@ -33,32 +49,11 @@ var authClient = new OktaAuth(config);
 
 Server-side web applications must also provide a [clientSecret](../README.md#clientsecret).
 
-## New Methods
-
-The following table shows a mapping of `authn` methods and the corresponding method (if any) in the `IDX` API.
-
-| authn | IDX |
-| ----  | ---- |
-| [signInWithCredentials](#signinwithcredentialsoptions) | [idx.authenticate](idx.md#idxauthenticate) |
-| [forgotPassword](#forgotpasswordoptions) | [idx.recoverPassword](#idxrecoverpassword) |
-| [unlockAccount](#unlockaccountoptions) | N/A |
-| [verifyRecoveryToken](#verifyrecoverytokenoptions) | N/A |
-| [session.setCookieAndRedirect](#sessionsetcookieandredirectsessiontoken-redirecturi) | N/A |
-| [transaction.cancel](#cancel) | [idx.cancel](#idxcancel) |
-
-## Transaction Model
+## Transaction Models
 
-Entrypoint methods in the `authn` API, such as [signInWithCredentials](#signinwithcredentialsoptions) and [forgotPassword](#forgotpasswordoptions) return a `Transaction` object which has a [status](#transactionstatus) field and various [methods](#common-methods) to proceed or cancel the transaction.  Methods in `IDX` API, such as [idx.authenticate](idx.md#idxauthenticate) and [idx.recoverPassword](#idxrecoverpassword) return an `IdxTransaction` object which also includes a [status](idx.md#status) field. If the transaction was successful and no further action is required, this value will be `IdxStatus.SUCCESS`. If the value is `IdxStatus.PENDING`, then some further action is required. The [nextStep](idx.md#nextstep) field will indicate which values are needed to continue the flow.
+### AuthN
```

Comment:
not sure if the N should be capitalized here. Do you see other examples of this style?

Reply:
It wasn't capitalized in other instances in this doc, I'll make the change